app/views/LivechatEditView.tsx

```diff
@@ -39,20 +42,65 @@ const styles = StyleSheet.create({
 	}
 });
 
-const Title = ({ title, theme }) =>
+interface ITitle {
+	title: string;
+	theme: string;
+}
+
+interface IField {
+	_id: string;
+	visibility: string;
+	scope: string;
+}
+
+interface IInputs {
+	[key: string]: string | string[] | undefined;
+	name: string;
+	email: string;
+	phone?: string;
+	topic: string;
+	tag: string[];
+}
+
+type TParams = IVisitor & IInputs;
+
+interface IInputsRefs {
+	name: RNTextInput | null;
+	phone: RNTextInput | null;
+	topic: RNTextInput | null;
+	[index: string]: RNTextInput | null;
```

Comment:
```js
interface IInputsRefs {
	[index: string]: RNTextInput | null;
	name: RNTextInput | null;
	phone: RNTextInput | null;
	topic: RNTextInput | null;
}
```

For some reason I prefer `index` to be the first one always, but that's on you :)